### PR TITLE
play music ui

### PR DESCRIPTION
**Pull request by victorbaxter**
## [Pull request title]
play music ui
### [Redmine link]
https://edu-redmine.framgia.vn/issues/7654
#### Screenshots (nếu có)
Screenshot_1552446055

### CODE REVIEW

**Comment by victorbaxter** [comment]
ready

File: app/src/main/res/drawable/ic_pause_button.xml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    android:viewportWidth="24"
+    android:viewportHeight="24"
+    android:width="24dp"
+    android:height="24dp">
+    <path
+        android:pathData="M12 2C6.48 2 2 6.48 2 12 2 17.52 6.48 22 12 22 17.52 22 22 17.52 22 12 22 6.48 17.52 2 12 2Zm-2 14l0 0C9.45 16 9 15.55 9 15L9 9C9 8.45 9.45 8 10 8l0 0c0.55 0 1 0.45 1 1l0 6c0 0.55 -0.45 1 -1 1zm4 0l0 0c-0.55 0 -1 -0.45 -1 -1l0 -6c0 -0.55 0.45 -1 1 -1l0 0c0.55 0 1 0.45 1 1l0 6c0 0.55 -0.45 1 -1 1z"
+        android:fillColor="#F55730" />
+</vector>
```
Comment (NguyenMinhTuanB) [inline app/src/main/res/drawable/ic_pause_button.xml]: need an endline

File: app/src/main/res/drawable/ic_play_button.xml [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    android:viewportWidth="24"
+    android:viewportHeight="24"
+    android:width="24dp"
+    android:height="24dp">
+    <path
+        android:pathData="M12 2C6.48 2 2 6.48 2 12 2 17.52 6.48 22 12 22 17.52 22 22 17.52 22 12 22 6.48 17.52 2 12 2Zm-2 13.5l0 -7c0 -0.41 0.47 -0.65 0.8 -0.4l4.67 3.5c0.27 0.2 0.27 0.6 0 0.8L10.8 15.9C10.47 16.15 10 15.91 10 15.5Z"
+        android:fillColor="#F55730" />
+
+</vector>
```
Comment (NguyenMinhTuanB) [inline app/src/main/res/drawable/ic_play_button.xml]: tương tự

File: app/src/main/res/layout/fragment_play_music.xml [hunk 3]
```diff
@@ -90,6 +91,93 @@
...
-    <!-- TODO: Update play music fragment layout -->
+    <android.support.constraint.ConstraintLayout
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_alignParentBottom="true"
+        android:background="@color/color_white">
+
+        <SeekBar
+            android:id="@+id/seekbar_duration"
+            android:layout_width="match_parent"
```
Comment (NguyenMinhTuanB) [inline app/src/main/res/layout/fragment_play_music.xml]: not match_parent

**Comment by victorbaxter** [comment]
ready

**Pull request merged**